frontend/workflows/kinesis/src/update-shard-count.tsx

```diff
@@ -55,17 +55,12 @@ const StreamDetails: React.FC<WizardChild> = () => {
     resourceData.updateData("targetShardCount", value);
   };
 
-  const values = [
-    Math.ceil(stream.currentShardCount * 0.5),
-    Math.ceil(stream.currentShardCount * 0.75),
-    Math.ceil(stream.currentShardCount * 1),
-    Math.ceil(stream.currentShardCount * 1.25),
-    Math.ceil(stream.currentShardCount * 1.5),
-    Math.ceil(stream.currentShardCount * 1.75),
-    Math.ceil(stream.currentShardCount * 2),
-  ];
-  const options = values.map(value => {
-    return { label: value.toString() };
+  // These will be used to multiply the currentShardCount, which will populate the choices
+  const multipliers = [0.5, 0.75, 1, 1.25, 1.5, 1.75, 2];
+
+  const values = multipliers.map(m => Math.ceil(stream.currentShardCount) * m);
+  const options = values.map((value, index) => {
+    return { label: `${value.toString()}(ceil of current * ${multipliers[index]})` };
```

Comment:
I wonder if we should do this or just uniq the list so we dont have duplicates, probably doesn't make a lot of sense to have the dupes anyways